### ISSUE TITLE
Undesired behavior when turning DatasetAnnotator on after setting a scene 

### ISSUE THREAD

**Issue by wematan** [BobbySolo/JTAMods#2]
Hi,

Tried to set a scene, this worked out pretty good, but when i try to record using DatasetAnnotator, 
right after pressing F8, the game shifts into some sort of odd scene where i only see body parts and part of the background and nothing happens.
trying to cancel the mods doesn't help.

**Comment by fabbrimatteo** [comment]
This is absolutely normal.

After pressing F8, the DatasetAnnotator mod waits some seconds before starting the registration. This is necessary because we want the game engine to spawn some "regular" pedestrians. This sleeping time is also needed because we want every people performing the "moving" task to start walking (we make them walk with a delay to avoid congestions).

In order to help the game engine to spawn "regular" pedestrians, i suggest you to define where the player has to wait before starting the registration. In the camera sub-menu, you can define "teleport 1" and "teleport 2" locations. Those locations has to be near the scenario, with the player turning the back to the scenario location. In this way, the game can spawn additional people without being seen.

So, just wait one minute, and the registration will start!

**Comment by wematan** [comment]
Thanks for the quick reply.

So just to make it clear, what is the proposed way to use JTA?

1. Teleport to wanted area
2. Set the scene by spawning peds 
3. Set the teleport values in the Camera option to the current scene location (why there are teleport 1 and 2?)
4. Press F8 and wait.

Another issue I encountered is i was not able to shut the data annotator off by CTRL+R it just keeps going until it gets to ~1800 frames.
how should i stop it?

And is there a way to keep the interactivity with the game in order to change the POV by yourself?

Thanks!

**Comment by fabbrimatteo** [comment]
The proposed way to use the mods is:

1. Move to the desired area
2. Prepare the scenario by spawning pedestrians
3. Move the player to the border of the scenario, turning the back to the scenario location. In this way, set up the two locations.
4. Move the player to a specific location (for non-moving sequences, the current camera position will be the camera position of your scenario).
5. Press "save new"
6. Press F8 to start recording the scenario (you can recording multiple scenario at one time)

You need two teleport location to maximize the probability to have additional game-spawned pedestrians.

For what concern the CTRL+R problem, have you followed the instructions of the ScriptHookV SDK readme under the section "Runtime asi script reloading"? 

Unfortunately, it is not possible to change the POV interactively during the registration. During the registration, the time slows down: this makes any use of the player impractical. Also, an interactivity feature is undesirable for us, because it implies that you have to be actively present during the registration (that can lasts days if you have many scenarios).
